tests/conftest.py

```diff
@@ -203,34 +203,37 @@ def event_loop() -> asyncio.AbstractEventLoop:  # type: ignore
 
 
 @pytest.fixture(scope="session")
-def client() -> SimpleCacheClient:
+def client() -> Iterator[SimpleCacheClient]:
     configuration = Laptop.latest()
     credential_provider = CredentialProvider.from_environment_variable("TEST_AUTH_TOKEN")
     with SimpleCacheClient(configuration, credential_provider, DEFAULT_TTL_SECONDS) as _client:
         # Ensure test cache exists
-        _client.create_cache(TEST_CACHE_NAME)
+        _client.create_cache(cast(str, TEST_CACHE_NAME))
```

Comment:
why is this cast necessary?

Reply:
We read `TEST_CACHE_NAME` from the environment variable, which is an `Optional[str]` (ie nullable). Mypy complains unless I do this, even if I assert the variable is not `None`

Reply:
boo.  :)  wonder if we could assign it to a different variable that was explicitly annotated as a string after we handle the env var check.  not a blocker.